<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackupDefintion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editSource"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editName"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbbType"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="editTarget"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblType">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblTarget">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblSource">
          <property name="text">
           <string>Source</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pbSelectSoruce">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxFilter">
        <property name="title">
         <string>Filter</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="4" column="2">
            <widget class="QLineEdit" name="editExcludeDirs"/>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="editIncludeFiles"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lblIncludedDirs_2">
             <property name="text">
              <string>Included Files</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lblExcludedDirs">
             <property name="text">
              <string>Excluded Directories</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblIncludedDirs">
             <property name="text">
              <string>Included Directories</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="lblExcludedDirs_2">
             <property name="text">
              <string>Excluded Files</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="editIncludeDirs"/>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="editExcludeFiles"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>cbbType</tabstop>
  <tabstop>editSource</tabstop>
  <tabstop>pbSelectSoruce</tabstop>
  <tabstop>editTarget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
